cmd/zrepl/mgmt_conn.c

```diff
@@ -587,20 +588,28 @@ uzfs_zvol_rebuild_dw_replica_start(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
 				uzfs_zinfo_drop_refcnt(zinfo, B_FALSE);
 				break;
 			}
+			uzfs_zvol_set_rebuild_status(zinfo->zv,
+			    ZVOL_REBUILDING_IN_PROGRESS);
 		} else {
 			uzfs_zinfo_take_refcnt(zinfo, B_FALSE);
 		}
 
 		io_sfd = create_and_bind("", B_FALSE, B_FALSE);
 		if (io_sfd < 0) {
-			printf("Rebuild IO socket create and bind failed\n");
+			/* Fail this rebuild process entirely */
+			ZREPL_ERRLOG("Rebuild IO socket create and bind "
```

Comment:
mgmt_conn.c has been cleaned from all ZREPL_ERRLOGs and we should not use them anymore. Logging to syslog is not good. You can replace it with standard fprintf(stderr,...) or printf.

Reply:
Logging comes with cost but it has its own advantage. I found logs helps a lot debugging in distributed and complex systems. But nonetheless, I have made change to use fprintf.